--- origsrc/hspell-1.0/Makefile.in	2006-04-18 07:48:39.000000000 -0500
+++ src/hspell-1.0/Makefile.in	2009-11-09 20:18:23.010361600 -0600
@@ -166,6 +168,9 @@ libhspell.a: dict_radix.o gimatria.o cor
 	ar cr $@ $^
 	-ranlib $@
 
+cyghspell-0.dll: dict_radix.o gimatria.o corlist.o libhspell.o $(EXTRAOBJECTS)
+	$(CC) -shared $(LDFLAGS) -Wl,--enable-auto-image-base,--out-implib,libhspell.dll.a -o $@ $^ $(LIBS)
+
 $(HSPELL_EXECUTABLE): hspell.o tclHash.o libhspell.a
 	$(CC) $(CFLAGS) $(LDFLAGS) -o $(HSPELL_EXECUTABLE) hspell.o tclHash.o libhspell.a $(LIBS)
 
