--- origsrc/hspell-1.0/mk_he_affix.c	2006-05-08 14:33:28.000000000 -0500
+++ src/hspell-1.0/mk_he_affix.c	2009-11-01 01:49:21.340808000 -0600
@@ -13,7 +13,7 @@
 #include "hspell.h"
 
 /* #define PREFIXFILE_COMMAND "gzip -dc hebrew.wgz.prefixes" */
-#define PREFIXFILE_COMMAND "gzip -dc hebrew.wgz.prefixes | specfilter"
+#define PREFIXFILE_COMMAND "gzip -dc hebrew.wgz.prefixes | ./specfilter"
 
 /* Convert a number in the range 0..52 (currently) to a readable character
    that can be used as the rule (prefix set) name. To facilitate merging our
